mod.ts

```diff
@@ -1,11 +1,16 @@
 import { botCache, Intents, startBot } from "./deps.ts";
 import { configs } from "./configs.ts";
-import { fileLoader, importDirectory } from "./src/utils/helpers.ts";
+import {
+  checkRequiredConfigVars,
+  fileLoader,
+  importDirectory,
+} from "./src/utils/helpers.ts";
 import { loadLanguages } from "./src/utils/i18next.ts";
 
 console.info(
   "Beginning Bot Startup Process. This can take a little bit depending on your system. Loading now...",
 );
+checkRequiredConfigVars();
```

Comment:
I'm not sure this is necessary seems like overkill. If a token isnt provided, it will just crash on first request.

Reply:
Yeah, I guess.
I can remove it, but I like my code to crash early. ^^